account/identity_blackbox_test.go

```diff
@@ -76,6 +76,23 @@ func (s *identityBlackBoxTest) TestOKToLoad() {
 	createAndLoad(s)
 }
 
+func (s *identityBlackBoxTest) TestOKExists() {
```

Comment:
Please make one test function `TestExists` and add subtests that test for existence and non-existence. That's more convenient.